### PR TITLE
fix(mirinae): do not make data from root_path as array at raw type of  dynamic layout

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [x] Not that difficult
- [ ] Approved feature branch merge to master

### CODE REVIEW

**Comment by stat-kwon** [comment]
Thanks for your quick action 👍

File: packages/mirinae/src/data-display/dynamic/dynamic-layout/templates/raw/index.vue [hunk 1]
```diff
@@ -60,8 +60,8 @@ export default {
             layoutName: computed(() => (props.options.translation_id ? vm.$t(props.options.translation_id) : props.name)),
             rootData: computed<any[]>(() => {
                 if (props.options.root_path) {
-                    const rootData = getValueByPath(props.data, props.options.root_path) ?? [];
-                    return Array.isArray(rootData) ? rootData : [rootData];
+                    const rootData = getValueByPath(props.data, props.options.root_path) ?? {};
+                    return rootData;
                 }
```
Comment (yuda110) [inline packages/mirinae/src/data-display/dynamic/dynamic-layout/templates/raw/index.vue:65]: nitpick: It seems like the type of `rootData` should be changed from `any[]` to `any`.

**Review by yuda110 — APPROVED** [review]
LGTM!! I left a comment!

**Review by stat-kwon — COMMENTED** [review]
Thanks for your quick action 👍

File: packages/mirinae/src/data-display/dynamic/dynamic-layout/templates/raw/index.vue (hunk 1 above)
Comment (WANZARGEN) [reply packages/mirinae/src/data-display/dynamic/dynamic-layout/templates/raw/index.vue:65]: Oh, I missed checking the type. Thanks!

**Pull request merged**